workitem_test.go

```diff
@@ -98,6 +105,7 @@ func TestCreateWI(t *testing.T) {
 }
 
 func TestListByFields(t *testing.T) {
+	resource.Require(t, resource.Database)
```

Comment:
I like `resource.Require` better because `Require` is a verb and therefore better suited for a function name.